be/src/olap/storage_engine.cpp

```diff
@@ -916,8 +916,13 @@ void StorageEngine::_parse_default_rowset_type() {
     boost::to_upper(default_rowset_type_config);
     if (default_rowset_type_config == "BETA") {
         _default_rowset_type = BETA_ROWSET;
-    } else {
+    } else if (default_rowset_type_config == "ALPHA") {
         _default_rowset_type = ALPHA_ROWSET;
+        LOG(WARNING) << "default_rowset_type in be.conf should be set to beta, alpha is not "
```

Comment:
I think we should use `CONF_Validator` to check this.
See `CONF_Validator` in be/src/common/config.h

Reply:
Not use validator because it will throw exception during bootstrap and the process could not start. It's bad for upgrade process.